src/test/java/uk/gov/di/ipv/cri/passport/lambda/PassportHandlerTest.java

```diff
@@ -32,10 +32,7 @@
 import java.security.spec.InvalidKeySpecException;
 import java.text.ParseException;
 import java.time.LocalDate;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.UUID;
+import java.util.*;
```

Comment:
Can we go back to the individual imports?
Intellij -> Preferences -> Editor -> Code Style -> Java -> Imports tab -> (Class|Names) count to use... set to 999

Reply:
Done